pyproject.toml

```diff
@@ -35,6 +35,7 @@ python = ">=3.9,<4"
 space_packet_parser = ">=4.2.0"
 spiceypy = ">=6.0.0"
 xarray = '>=2023.0.0'
+hermes_core = {git = "https://github.com/HERMES-SOC/hermes_core.git", branch="main"}
```

Comment:
The HERMES version is more stable for now, but I'm moving a lot of functionality to the https://github.com/swxsoc/swxsoc repo. The `SWXDataSchema` is the "generalized" version that does not have the hermes-specific attributes and adds in the layering niceties. This is the class/version that I would like to split out into its own repo, maybe something like `swxschema`. This repo might be volatile in the upcoming weeks/months as I'm extending the schema for FITS/SOLARNET metadata attributes so we should discuss whether we want to do the division before or after I want to make a lot of volatile changes.